pageComponents/NewPostPage/NewPostPage.tsx

```diff
@@ -0,0 +1,284 @@
+
+          <HorizontalLabel>
+            <HorizontalItemTitle>Big:</HorizontalItemTitle>
+            <BetSizeInput defaultValue={1} disabled />
+          </HorizontalLabel>
+
+          <HorizontalLabel>
+            <HorizontalItemTitle>Anti:</HorizontalItemTitle>
+            <BetSizeInput defaultValue={0} />
+          </HorizontalLabel>
+        </Blinds>
+
+        <SectionTitle>Your cards and position</SectionTitle>
+        <HeroInfo>
+          <HorizontalLabel>
+            <HorizontalItemTitle>Hand:</HorizontalItemTitle>
+            <HeroHand>
+              <_SelectablePlayingCard />
+              <_SelectablePlayingCard />
+            </HeroHand>
+          </HorizontalLabel>
+
+          <HorizontalLabel>
+            <HorizontalItemTitle>at:</HorizontalItemTitle>
+            <HeroPositionSelect playerLength={2} />
+          </HorizontalLabel>
+        </HeroInfo>
+
+        <SectionTitle>Preflop</SectionTitle>
+        <StreetActionInput
+          playerLength={post.playerLength}
+          actions={post.preflop}
+        />
+
+        {0 < post.flop.length ? (
+          <>
+            <SectionTitle>Flop</SectionTitle>
+            <StreetActionInput
+              playerLength={post.playerLength}
+              actions={post.flop}
+            />
+          </>
+        ) : null}
+
+        {0 < post.flop.length ? (
+          <>
+            <SectionTitle>Turn</SectionTitle>
+            <StreetActionInput
+              playerLength={post.playerLength}
+              actions={post.turn}
+            />
+          </>
+        ) : null}
+
+        {0 < post.flop.length ? (
+          <>
+            <SectionTitle>River</SectionTitle>
+            <StreetActionInput
+              playerLength={post.playerLength}
+              actions={post.river}
+            />
+          </>
+        ) : null}
+
+        {0 < post.showdownActivePlayer.length ? (
+          <>
+            <SectionTitle>Showdown</SectionTitle>
+            <PlayerHandSelectors
+              playerLength={post.playerLength}
+              playerHands={post.playerHands}
+              activePlayerIndexes={post.showdownActivePlayer}
+            />
+          </>
+        ) : null}
+
+        <Actions>
+          <Button
+            variant={ButtonVariant.secondary}
+            onClick={() => {
+              router.replace("/new", "/");
+            }}
+          >
+            Cancel
+          </Button>
+          <Button>Leave a Post</Button>
+        </Actions>
+      </Content>
+    </Root>
+  );
+}
+
+const Root = styled.div`
+  display: grid;
+  min-width: 375px;
+  grid-template-columns: minmax(64px, auto) minmax(375px, 1024px) minmax(
+      64px,
+      auto
+    );
+  grid-template-areas: "header header header" ". content .";
+`;
+
+const _HeadBar = styled(HeadBar)`
+  grid-area: header;
+`;
+
+const Content = styled.section`
+  grid-area: content;
+  padding: 64px 0 128px;
+`;
+
+const Headline = styled.h2`
+  margin-bottom: 32px;
```

Comment:
I think it would be better 🤔

```suggestion
  margin-bottom: 28px;
```

<img width="1078" alt="Screen Shot 2020-06-15 at 7 23 14 PM" src="https://user-images.githubusercontent.com/4289883/84724712-b1dae500-af3d-11ea-92d3-e66926242b2b.png">